src/doom/d_main.c

```diff
@@ -1493,6 +1528,11 @@ void D_DoomMain (void)
     {
         gamevariant = bfgedition;
     }
+    else if (gamemission == doom2 && W_CheckNumForName("M_RDTHIS") < 0 && W_CheckNumForName("M_EPISOD") < 0 && W_CheckNumForName("M_EPI1") < 0
```

Comment:
I think it would be better if we were consistent with the behavior when using `chex.wad` and `hacx.wad`, which is that we identify by IWAD name. Can you do that instead?

In fact I wonder if we need the new gamevariant at all since we do not change any of the game logic - we merely load a dehacked patch.

Reply:
I'll look into revisions. No, we don't really need a new game version per se, I just did it that way to flow with the rest of the code.

Reply:
Okay, let's stick with the lump checking for now - all it really does is load the dehacked patch, after all.